projects/storefrontlib/src/cms-components/cart/add-to-cart/added-to-cart-dialog/added-to-cart-dialog.component.ts

```diff
@@ -60,6 +62,7 @@ export class AddedToCartDialogComponent implements OnInit {
 
   updateEntry({ item, updatedQuantity }): void {
     this.cartService.updateEntry(item.entryNumber, updatedQuantity);
+    this.modalIsOpen = true;
```

Comment:
This line would be overwritten everytime this method is called, and it's unnecessary to overwrite this value, which we need to set to true only once. I believe it could be done in this.entry$, after part with `this.form.markAsPristine();`